src/components/UserFeedbackComponent.jsx

```diff
@@ -74,6 +75,13 @@ export class UserFeedbackComponent extends React.Component {
     return feedbackRecord;
   }
 
+  renderTableHeaders = () => {
```

Comment:
@3m3kalionel I usually discourage this pattern as it has the tendency of making a file be too long in the future. It encourages devs to keep on adding more methods in the file, instead of them evaluating if they need another component. Ideally, what we want is to only have the markup rendered by the `render` function. A good case is what you have just done as this is the pattern that has been used in the file previously. Please use the markup directly where the method is being used. E.g.
```
<TableHeaderComponent
  titles={[
    'Submitted by',
    'Date Submitted',
    'Type',
    'Message',
    'Action'
  ]}
/>
```

Reply:
Thanks for explaining the reason. This actually is better.